package.json

```diff
@@ -27,52 +27,45 @@
     ]
   },
   "devDependencies": {
-    "@cloudflare/workers-types": "^3.14.1",
-    "@semantic-release/changelog": "^6.0.1",
-    "@semantic-release/commit-analyzer": "^9.0.2",
-    "@semantic-release/git": "^10.0.1",
-    "@semantic-release/github": "^8.0.5",
-    "@semantic-release/npm": "^9.0.1",
-    "@semantic-release/release-notes-generator": "^10.0.3",
-    "@types/jest": "^27.5.2",
+    "@cloudflare/workers-types": "^3.11.0",
+    "@types/jest": "^29.0.0",
     "@types/service-worker-mock": "^2.0.1",
-    "@typescript-eslint/eslint-plugin": "^4.33.0",
-    "@typescript-eslint/parser": "^4.33.0",
-    "conventional-changelog-conventionalcommits": "^5.0.0",
-    "eslint": "^7.32.0",
+    "@typescript-eslint/eslint-plugin": "^5.36.2",
+    "@typescript-eslint/parser": "^5.36.2",
+    "eslint": "^8.23.0",
     "eslint-config-prettier": "^8.5.0",
     "eslint-config-typescript": "^3.0.0",
-    "jest": "^27.5.1",
-    "miniflare": "^2.7.1",
-    "prettier": "^2.7.1",
-    "semantic-release": "^19.0.5",
+    "jest": "^29.0.2",
+    "miniflare": "^2.4.0",
+    "prettier": "^2.3.0",
     "service-worker-mock": "^2.0.5",
-    "terser-webpack-plugin": "^5.3.5",
-    "ts-jest": "^27.1.5",
-    "ts-loader": "^9.3.1",
-    "typescript": "^4.7.4",
-    "webpack": "^5.74.0",
-    "webpack-cli": "^4.10.0"
-  },
-  "dependencies": {
-    "@cosmjs/amino": "^0.28.4",
-    "@cosmjs/encoding": "^0.28.4",
-    "@cosmjs/proto-signing": "^0.28.4",
-    "@cosmjs/stargate": "^0.28.13",
-    "@lum-network/sdk-javascript": "^0.7.6",
+    "terser-webpack-plugin": "^5.3.1",
+    "ts-jest": "29.0.0-next.1",
+    "ts-loader": "^9.2.2",
+    "typescript": "^4.3.2",
+    "webpack": "^5.38.1",
+    "webpack-bundle-analyzer": "^4.6.1",
+    "webpack-cli": "^4.7.0",
     "browserify-fs": "^1.0.0",
-    "buffer": "6.0.3",
     "crypto-browserify": "3.12.0",
-    "itty-router": "^2.6.1",
-    "multibase": "4.0.6",
-    "multicodec": "3.2.1",
     "path-browserify": "1.0.1",
-    "process": "^0.11.10",
+    "buffer": "6.0.3",
+	"eslint-plugin-jest": "^27.0.1",
+	"eslint-plugin-prettier": "^4.2.1",
     "stream-browserify": "3.0.0",
+    "process": "^0.11.10",
     "util": "^0.12.4"
   },
-  "publishConfig": {
```

Comment:
@jay-dee7 We probably need this to configure the registry scope as well on release.

Reply:
totally accidental